inspirehep/modules/workflows/tasks/arxiv.py

```diff
@@ -122,7 +122,7 @@ def arxiv_plot_extract(obj, eng):
             return
 
         for idx, plot in enumerate(plots):
-            obj.files[plot.get('name')] = BytesIO(open(plot.get('url')))
```

Comment:
Hmm... the only thing I see makes sense to use a bytesio would be:
```python
obj.files[plot.get('name')] = BytesIO(open(plot.get('url'), 'rb').read())
```

Maybe it's one of those python2<->python3 thingies...

btw. I think that you should not use 'get' here, just old `['mykey']` so it  does not silently fail